src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -51,6 +51,13 @@ function AdditionalMetadataFields(props) {
           disabled={disabled}
           required={false}
         />
+        <Field
+          name="additional_metadata.course_term_override"
+          component={RenderInputTextField}
+          label=<FieldLabel id="course_term_override.label" text="Override the course term" />
```

Comment:
nit: text should be "Course Term Override". Also add its info on field info tip to explain what that actually is.